### PR TITLE
BIPT 66 - Approve, Reject, Abstain buttons on the voting component

### PR DESCRIPTION
**Pull request by kinjalmudgal89**
### Related Ticket

<https://eyblockchain.atlassian.net/browse/__TICKET__>

### QA Instructions

- [ ] Step 1

### Checklist

- [x] I have reviewed the code changes myself
- [x] My code meets all of the acceptance criteria of the ticket it closes.
- [x] I have removed unused code (e.g., `console.logs`, commented out blocks, etc.)
- [ ] I have made all necessary changes to the documentation.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] Any dependent changes have been merged and published.
- [x] Check PR title name is correct.

### CODE REVIEW

**Comment by abdulmalik97** [comment]
@kinjalmudgal89  Is there a reason why we are creating a PR for only a UI change which is functionally incomplete? 

According to me it's better to have the full functionality of voting built and then pushed with a PR.

**Comment by kinjalmudgal89** [comment]
> @kinjalmudgal89 Is there a reason why we are creating a PR for only a UI change which is functionally incomplete?
> 
> According to me it's better to have the full functionality of voting built and then pushed with a PR.

@abdulmalik97 So i did speak with Nikhil regarding the voting feature, he wanted me to create different PRs for different tickets and not merge tickets as it becomes difficult to review big chunk of codes. BIPT 66 is only to show the voting buttons on the component. Hence i created a new branch and PR for this ticket.

**Comment by abdulmalik97** [comment]
@kinjalmudgal89  Okay, that makes sense. Thanks for the clarification!

**Review by abdulmalik97 — APPROVED** [review]

**Review by Lana-Butorovic — APPROVED** [review]

File: components/ProposalRow.tsx [hunk 1]
```diff
@@ -15,10 +15,11 @@ import {
   AccordionIcon,
   IconButton,
   Button,
+  Input,
 } from "@chakra-ui/react";
 
-import { EditIcon } from "@chakra-ui/icons";
-import { Proposal, ProposalStatus, RFCStatus } from "@prisma/client";
+import { EditIcon, } from "@chakra-ui/icons";
```
Comment (nagrawal3) [inline components/ProposalRow.tsx:21]: extra comma

**Review by nagrawal3 — COMMENTED** [review]
your code looks good, but for consistency, I would suggest running a linter on it. There are lot of blank lines, and uneven spacing.

**Review by meaghanfitzgerald — APPROVED** [review]
Please format the file for spacing and notation and then we can merge!

**Comment by Lana-Butorovic** [comment]
@nagrawal3 @meaghanfitzgerald @kinjalmudgal89 ran linting on the file, should be good to merge :)

**Pull request merged**